src/main/java/AppScreen.java

```diff
@@ -0,0 +1,119 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+
+
+public class AppScreen implements AppScreenPresenter, AppScreenController {
+
+    final JFrame JFRAME;
+    final User CURR_USER;
```

Comment:
According to [this StackOverflow thread](https://stackoverflow.com/questions/27660741/are-getters-required-when-the-field-vars-are-final), only static final variables have all capitalization.

Reply:
Oh really? I always thought variables just needed to be final to use all capitalization - I'll probably make them static then